client/starwhale/utils/venv.py

```diff
@@ -549,16 +549,33 @@ def activate_python_env(
     mode: str,
     identity: str,
 ) -> None:
-    # TODO: switch shell python environment directly
-    console.print(":cake: run command in shell :cake:")
-
     if mode == PythonRunEnv.VENV:
         cmd = f"source {identity}/bin/activate"
     elif mode == PythonRunEnv.CONDA:
         cmd = f"conda activate {identity}"
     else:
         raise NoSupportError(mode)
 
+    shell = os.getenv("SHELL")
```

Comment:
use `shellingham` to detect shell?  example: https://github.com/star-whale/starwhale/blob/main/client/starwhale/utils/__init__.py#L184

Reply:
done